packages/e2e-test-utils/src/press-key-times.js

```diff
@@ -1,31 +1,11 @@
-/**
- * External dependencies
- */
-import { times } from 'lodash';
-
-/**
- * Given an array of functions, each returning a promise, performs all
- * promises in sequence (waterfall) order.
- *
- * @param {Function[]} sequence Array of promise creators.
- *
- * @return {Promise} Promise resolving once all in the sequence complete.
- */
-async function promiseSequence( sequence ) {
-	return sequence.reduce(
-		( current, next ) => current.then( next ),
-		Promise.resolve()
-	);
-}
-
 /**
  * Presses the given keyboard key a number of times in sequence.
  *
  * @param {string} key   Key to press.
  * @param {number} count Number of times to press.
- *
- * @return {Promise} Promise resolving when key presses complete.
  */
 export async function pressKeyTimes( key, count ) {
-	return promiseSequence( times( count, () => () => page.keyboard.press( key ) ) );
```

Comment:
Was it determine whether this change was strictly necessary? It still seems to me like the previous implementation should work fine, though I'm in a similar position of questioning whether a test I'm seeing failures for might be failing at this function.

Reply:
It didn't help at all 😅  I can revert when recommended. I personally find it the current implementation much easier to follow.

Reply:
> It didn't help at all 😅 I can revert when recommended. I personally find it the current implementation much easier to follow.

I agree and I don't mind the refactor, but for my own purposes I wasn't sure if it makes any difference. The one reluctance I might have is that we lose the `docgen`-generated `Promise` return value in the current set of changes.

Reply:
Added `return` back in 644b0bc.